boot/bootutil/src/ed25519_psa.c

```diff
@@ -25,11 +25,20 @@ BOOT_LOG_MODULE_REGISTER(ed25519_psa);
 
 #if defined(CONFIG_BOOT_SIGNATURE_USING_KMU)
 /* List of KMU stored key ids available for MCUboot */
+#define PSA_KEY_INDEX_SIZE 2
+
+#if CONFIG_MCUBOOT_MCUBOOT_IMAGE_NUMBER != -1 || \
+	defined(CONFIG_NCS_BOOT_SIGNATURE_KMU_ORIGINAL_MAPPING)
+#define PSA_KEY_STARTING_ID 226
```

Comment:
Should we maybe use Kconfig for KMU base address?

Reply:
would probably make sense and allow for future devices if they have different mappings, but will leave for someone else to implement